### DISCUSSION TITLE
On Page Swipe listener availalbe ?

### DISCUSSION THREAD

**Discussion by girubhai** [AppIntro/AppIntro#1096] [General]
If User is on last slide and do Swipe right then we want to do task as done button i.e. open next screen. Is there any method available to do the same when swipe right on last slide ? Have checked the **AppIntroBase.kt** class but dinn't find the particular method. had I missed something ? Thanks in advance.

**Comment by cortinico** [comment]
> Is there any method available to do the same when swipe right on last slide ?

Nope sadly this is not supported.

**Reply by girubhai** [reply]
Thanks to prompt response. Is there any chance to include this in future ?

**Reply by cortinico** [reply]
We're open to reviewing pull requests :)
